.github/workflows/release-docker-image.yaml

```
@@ -103,6 +116,25 @@ jobs:
               }'            
           fi
 
+  associate-commits:
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    if: ${{ secrets.sentry-auth-token }} != ''
```

I think brackets are not needed here.

```suggestion
    if: secrets.sentry-auth-token != ''
```